python/doc/theory/probabilistic_modeling/parametric_spectral_model.rst

```diff
@@ -22,15 +22,16 @@ Parametric spectral density functions
 
 **The Cauchy spectral model**
 
-Its is associated to the Exponential covariance model. The Cauchy spectral model is defined by:
+Its is associated to the Kronecker covariance model built upon an exponential covariance model (AbsoluteExponential). The Cauchy spectral model is defined by:
 
 .. math::
   :label: cauchyModel
 
-    S_{ij}(f) = \displaystyle \frac{4R_{ij}a_ia_j(\lambda_i+ \lambda_j)}{(\lambda_i+ \lambda_j)^2 + (4\pi f)^2}, \quad \forall (i,j) \leq d
+    S_{ij}(f) = \displaystyle 2 \Sigma_{ij}\prod_{k=1}^{n} \frac{\theta_k}{1 + (2\pi \theta_k f)^2}, \quad \forall (i,j) \leq d
```

Comment:
```suggestion
    [S(f)]_{ij} = \displaystyle 2 \mat{\Sigma}_{ij}\prod_{k=1}^{n} \frac{\theta_k}{1 + (2\pi \theta_k f)^2}, \quad \forall (i,j) \leq d
```
I think this notation is clearer: S is defined in the introduction, but not S_{ij}. Also, Sigma needs to be consistently written as a matrix.